packages/grid/_modules_/grid/hooks/root/useApi.ts

```diff
@@ -5,13 +5,17 @@ import { useLogger } from '../utils/useLogger';
 import { GRID_COMPONENT_ERROR, GRID_UNMOUNT } from '../../constants/eventsConstants';
 import { useGridApiMethod } from './useGridApiMethod';
 import { MuiEvent } from '../../models/gridOptions';
+import { Signature } from './useGridApiEventHandler';
+import { useGridSelector } from '../features/core/useGridSelector';
+import { optionsSelector } from '../utils/optionsSelector';
 
 const isSynthenticEvent = (event: any): event is React.SyntheticEvent => {
   return event.isPropagationStopped !== undefined;
 };
 
 export function useApi(apiRef: GridApiRef): void {
```

Comment:
```suggestion
export function useApi(apiRef: GridApiRef, { signature }: Pick<GridComponentProps, 'signature'>): void {
```

Reply:
I'll keep it as `props` based on this discussion https://github.com/mui-org/material-ui-x/pull/2275#discussion_r682581815